ivy/functional/backends/paddle/device.py

```diff
@@ -8,71 +8,118 @@
 import ivy
 from ivy.functional.ivy.device import Profiler as BaseProfiler
 from paddle.fluid.libpaddle import Place
-
+from paddle.fluid import core
```

Comment:
please note that the `paddle.fluid` namespace and API is going to be deprecated soon, I'll try to find alternatives and suggest changes